src/scss/_header.scss

```diff
@@ -179,7 +179,7 @@
                 &:hover {
                     a,
                     .dropdown-title {
-                        @apply opacity-50;
+                        color: #7b61ff;
```

Comment:
A couple questions here - I got this color from the [figma](https://www.figma.com/file/AVYzxAC8g4gTOabh2RqfQx/Marketing-Site-Navigation?node-id=306%3A1799) design. I noticed we currently don't have this color anywhere in the site, so I just hard coded it here. Should I make another variable specifically for this color, and if so where should that be placed? _colors.scss seems to only contain official brand colors, so didn't want to add it there. Or should I just use one of the brand colors instead (I tried some of the brand colors, but this one looks a lot better since they didn't have the "pop" that this color had and looked a bit faded by comparison).

Reply:
@sean1588 @jonelleb i believe this should be the exact same color as the primary buttons, right?

Reply:
but if not, yah lets get this added in with our brand colors

Reply:
The primary buttons seem to use the violet brand color so I'll get this color added. thanks @susanev !!

Reply:
k, Jonelle is done for the day, but def sync up with her monday anything she says she override me

Reply:
@sean1588 Sorry about that, I thought that I was using the same color as the purple primary buttons that we have on our site, but if not, if we can get this color added that would be great. Thanks!

Reply:
if the intention was to use the same color as the primary buttons, lets go with that, because `#7b61ff` fails contrast, but the primary button color of: `#805ac3` does not.

Reply:
That's works for me. Sorry for the confusion.